test/E2ETest/DockerAssert.cs

```diff
@@ -46,6 +47,16 @@ void OnOutput(string text)
             }
         }
 
+        public static Task AssertAllContainersExistAsync(ITestOutputHelper output, string[] containers)
+            => AssertAboutContainersAsync(output, containers,
+                (existing, given) => given.All(c => existing.Contains(c)),
+                given => $"Not all containers ({string.Join(", ", given)}) exist");
```

Comment:
This would be better if it could print the list of containers that were still running, instead of the full set.